version-map.txt

```diff
@@ -1,2 +1,2 @@
 # List of Sopel versions to build:
-v7.0.0
+v7.0.2
```

Comment:
```suggestion
# remove v7.0.1 in next version bump
v7.0.1
v7.0.2
```

I would like all versions of Sopel to be tagged/exist in the public Docker repo. Since I missed `v7.0.1`, lets add it here and have that image get built at least once (and leave a comment to remove it for the next version bump). One commit with both versions added, or two separate ones adding `.0.1` first then `.0.2` are fine.

Reply:
Implementing this suggestion can use multiple commits, but in general, can you squash all commits related to a single version bump into just one commit (see repo commit history for message format)? 

Thanks in advance!